src/utils/request.ts

```diff
@@ -14,3 +28,82 @@ export function parseUrl(request: Request): URL | undefined {
 
   return url;
 }
+
+export function parseConditionalHeaders(headers: Headers): ConditionalHeaders {
+  const ifModifiedSince = headers.has('if-modified-since')
+    ? new Date(headers.get('if-modified-since')!)
+    : undefined;
+  if (ifModifiedSince !== undefined) {
+    ifModifiedSince.setSeconds(ifModifiedSince.getSeconds() + 1);
+  }
+
+  return {
+    ifMatch: headers.has('if-match')
```

Comment:
Can you move all these assignments to dedicated constants above, so that the return statement only contains the values?